src/settings.rs

```diff
@@ -17,8 +18,22 @@ pub struct DnsSettings {
 
 impl DnsSettings {
   pub fn load(filename: String) -> Result<Self, Box<dyn Error>> {
-    let contents = fs::read_to_string(shellexpand::full(filename.as_str()).unwrap().to_string())
-      .expect("Aw man, there was an issue while opening the config file :(");
+
+    let filenames= [filename.as_str(), "./dns.config.yaml", "~/.config/simpledns/dns.config.yaml", "/etc/simpledns/dns.config.yaml"];
+    let mut matched = false;
+    let mut contents = String::new();
+    let mut config_file = String::new();
+    for filename in filenames {    
+      contents = match fs::read_to_string(shellexpand::full(filename).unwrap().to_string()) {
+        Ok(data) => { matched = true; config_file = String::from(filename); data},
+        Err(_) => { continue },
+      };
+      break
+    } 
+    if !matched { panic!("Aw man, there was an issue while opening the config file :(") }
```

Comment:
We can create a default file instead of panicking actually! Then also log that it was created and the default settings so they can know to change what they want from it.

Then the user doesnt have to go and manually create the file themselves.

Reply:
I am rethinking this comment cause the `init` command initializes the database too and if we just run `simpledns start` it will panic #17

Reply:
Okay, how would you feel about adding 'create a config file as necessary' to #17? That would let us move the other changes into main.

Reply:
Yeah lets do that. Can you add a comment on that issue?